bot_messages.py

```diff
@@ -1,4 +1,13 @@
-#slackbot tutorial original code from https://github.com/slackapi/python-slack-sdk/tree/main/tutorial, modified.
+# This file contains all of the messages that the Slack Bot sends in channels.
+# slackbot tutorial original code from https://github.com/slackapi/python-slack-sdk/tree/main/tutorial, modified.
+
+# questions that the bot asks when documenting
+question_one = "What do you want to title this wiki?"
+question_two = "Do you want to upload a supplemental image? If so, upload a link of the image. If not, type \"None\"."
+question_three = "Do you want to add anything else? If so, type any supplemental information you wish to add. If not, type \"None\"."
```

Comment:
I should have caught this in the first review, but you could also make this an array indexed by your last_question_asked variable - totally no pressure to make this change in this PR though!